src/audio/igo_nr/igo_nr.c

```diff
@@ -0,0 +1,602 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
check patch is still complaining about these comments

Reply:
checkpatch also complain even if it is changed to `/* */`.  

`WARNING: Improper SPDX comment style for 'src/audio/igo_nr/igo_nr.c', please use '//' instead`
`#11: FILE: src/audio/igo_nr/igo_nr.c:1:`
`+/*SPDX-License-Identifier: BSD-3-Clause`

`WARNING: Missing or malformed SPDX-License-Identifier tag in line 1`
`#11: FILE: src/audio/igo_nr/igo_nr.c:1:`
`+/*SPDX-License-Identifier: BSD-3-Clause`

so I take other components as reference and remain `//` .

Reply:
You need a space between `/*` and your first character. That is why it is upset in your second case.

Reply:
I get same warning even with that space added. Let me try to force push here to see if the result is same or not.

Reply:
turns out checkpatch and post commit script don't agree with each other. Use `//` sorry for the noise. @lgirdwood should we fix the post commit hook?

Reply:
@cujomalainey it's okay. guess that's why everyone is using `//`.

Reply:
yep, checkpatch is very Linux specific and should be ignored here. It does need some TLC as it also brings up requests that break XCC.  @marc-hb fyi - when time permits.

Reply:
I recently added `--ignore C99_COMMENT_TOLERANCE` in commit eb459078f302 because I got tired of bumping into this unwritten SOF policy and wasting precious code review time. As you've noticed, this is in contradiction with another checkpatch warning. Checkpatch is not perfect and is expected to have some contradictions.

https://www.kernel.org/doc/html/v4.12/process/5.Posting.html#sending-the-patch
> Are you sure your patch is free of silly mistakes? You should always run patches through scripts/checkpatch.pl and address the complaints it comes up with. Please bear in mind that **checkpatch.pl**, while being the embodiment of a fair amount of thought about what kernel patches should look like, **is not smarter than you.** **If fixing a checkpatch.pl complaint would make the code worse, don’t do it.**


Here's another, much more painful contradiction:

```
src/test.c:6: WARNING: break quoted strings at a space character
+	f("very long line very long line very long line very long line"
+	  "very long line very long line very long line very long line");
```
but:
```
src/test.c:5: WARNING: line length of 133 exceeds 100 columns
+	f("very long line very long line very long line very long line very long line very long line very long line very long line");
```
... and `--strict` is not even turned on for this one. Any way out of it?

As long as these contradictions are _rare_ then everything fine. We don't add new files every day whereas any line of code could have a C99 comment.

If something becomes too frequent then we go and fix it: https://github.com/thesofproject/sof/commits/main/scripts/checkpatch.pl